src/Thentos/Util.hs

```diff
@@ -77,3 +83,13 @@ readsPrecEnumBoundedShow _ s = f [minBound..]
         (s0, s1) -> if s0 == s' then [(x, s1)] else f xs
       where
         s' = show x
+
+
```

Comment:
(-:

I would prefer to keep `<//>` in the lib.  It should always be used instead of `<>` used for concatenizationalizing paths, so we don't have to check manually for leading or trailing slashes.